tests/gateways/test_jlap.py

```diff
@@ -0,0 +1,479 @@
+# Copyright (C) 2012 Anaconda, Inc
```

Comment:
Could we split this file up a little by topic? E.g. `jlap/test_reader.py`, `jlap/test_client.py` and `jlap/test_interface.py`?

Reply:
Maybe? I wrote this by expanding individual test functions until 100% code coverage or until it made more sense to open a new function.

Reply:
Totally get that, just trying to reduce the file length and sort some of the tests in a way that lends itself to overview for less experienced devs